chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java

```diff
@@ -293,6 +293,7 @@ public PermissionDeletedResponse deleteUserFromDeveloper(
             toCreate.setDeveloperCode(developerInfo.getDeveloper().getDeveloperCode());
             toCreate.setName(developerInfo.getDeveloper().getName());
             toCreate.setWebsite(developerInfo.getDeveloper().getWebsite());
+            toCreate.setSelfDeveloper(developerInfo.getDeveloper().getSelfDeveloper());
```

Comment:
I think there should be an additional change in this file... in the split method?